src/main/java/io/github/syst3ms/skriptparser/expressions/ExprStringChars.java

```diff
@@ -0,0 +1,75 @@
+package io.github.syst3ms.skriptparser.expressions;
+
+import java.util.function.Function;
+
+import org.jetbrains.annotations.Nullable;
+
+import io.github.syst3ms.skriptparser.Parser;
+import io.github.syst3ms.skriptparser.lang.Expression;
+import io.github.syst3ms.skriptparser.lang.TriggerContext;
+import io.github.syst3ms.skriptparser.parsing.ParseContext;
+import io.github.syst3ms.skriptparser.registration.PatternInfos;
+
+/**
+ * All uppercase, lowercase, or digit characters in a string.
+ *
+ * @name Characters
+ * @pattern [all [of the]] upper[ ]case char[acter]s in %strings%
+ * @pattern [all [of the]] lower[ ]case char[acter]s in %strings%
+ * @pattern [all [of the]] digit char[acter]s in %strings%
+ * @pattern [all [of the]] special char[acter]s in %strings%
+ * @pattern [all [of the]] [white[]]space char[acter]s in %strings%
+ * @since ALPHA
+ * @author Olyno
+ */
+public class ExprStringChars implements Expression<String> {
+
+    private enum CharType {
```

Comment:
I know I'm not really one to talk, but if I'm not mistaken, this isn't proper ordering of the elements inside a class.